WordPressLoginFlow/src/main/java/org/wordpress/android/login/LoginBaseFormFragment.java

```diff
@@ -243,6 +243,11 @@ protected void startProgress(boolean cancellable) {
             mBottomButton.setEnabled(false);
         }
 
+        if (mProgressDialog != null) {
+            mProgressDialog.setOnCancelListener(null);
+            mProgressDialog.cancel();
+        }
```

Comment:
This fixes a bug when using hardware keyboards (like in emulators) that result in stacking two progress dialogs and dismissing just one of them when the login is done:


https://github.com/user-attachments/assets/86e0e6c8-97e4-4c66-9698-1a8650d2908b